.github/workflows/system-tests.yml

```diff
@@ -84,11 +84,10 @@ jobs:
           path: artifact.tar.gz
 
   parametric:
-    needs:
-      - build-artifacts
```

Comment:
why doesn't it need build anymore ?

Reply:
It probably does, this was removed by mistake.